lib/core/src/restore.rs

```diff
@@ -465,8 +547,14 @@ impl LiquidSdk {
                         false => (Some(second_tx_id), Some(first_tx_id)),
                     }
                 }
+                0 => {
+                    debug!(
+                        "No BTC script history found while recovering data for Chain Receive Swap {swap_id}"
+                    );
+                    (None, None)
+                }
```

Comment:
```suggestion
```

See comment above about `0` case being already handled in `n` branch.

Reply:
Same as above